migration/src/m0000250_create_sbom_package.rs

```diff
@@ -14,6 +14,8 @@ impl MigrationTrait for Migration {
                     .if_not_exists()
                     .col(ColumnDef::new(SbomPackage::SbomId).uuid().not_null())
                     .col(ColumnDef::new(SbomPackage::NodeId).string().not_null())
+                    .col(ColumnDef::new(SbomPackage::Group).string())
```

Comment:
We do not modify migrations, but create new ones. @mrizzi can help you with that.